modules/backup-and-restore/pages/configurations.adoc

```diff
@@ -171,6 +193,18 @@ Alternatively, instead of using `AccessKeyID` and `SecretAccessKey`, you may use
 ----
 gadmin config set "System.Backup.S3.RoleARN" "arn:aws:iam::account:role/role-name-with-path"
 ----
+
+. Set the AWS Region for EC2 instances:
+
+If you're running TigerGraph on an EC2 instance, set the AWS region for the AWS SDK by configuring the region as an environment variable:
++
+[console,]
+----
+gadmin config set Executor.BasicConfig.Env "AWS_REGION=us-west-2;$(gadmin config get Executor.BasicConfig.Env)"
+gadmin restart exe -y
```

Comment:
You have 
`gadmin restart`
then
`gadmin config apply`

If you think back to some of the other configuration tickets you have worked on, the order is
```
gadmin config apply
gadmin restart
```